doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -379,63 +377,77 @@ A Single-Stage Problem
+  :skipif: not (baron.available() and baron.license_is_valid())
 
   >>> # === Designate which variables correspond to first- and second-stage degrees of freedom ===
   >>> first_stage_variables =[m.x1, m.x2, m.x3, m.x4, m.x5, m.x6,
-                        m.x19, m.x20, m.x21, m.x22, m.x23, m.x24, m.x31]
+  ...                      m.x19, m.x20, m.x21, m.x22, m.x23, m.x24, m.x31]
   >>> second_stage_variables = []
   >>> # The remaining variables are implicitly designated to be state variables
 
   >>> # === Call PyROS to solve the robust optimization problem ===
-  >>> results_1 = pyros_solver.solve(model = m,
-                                    first_stage_variables = first_stage_variables,
-                                    second_stage_variables = second_stage_variables,
-                                    uncertain_params = uncertain_parameters,
-                                    uncertainty_set = box_uncertainty_set,
-                                    local_solver = local_solver,
-                                    global_solver= global_solver,
-                                    options = {
-                                       "objective_focus": pyros.ObjectiveType.worst_case,
-                                       "solve_master_globally": True
-                                     })
+  >>> results_1 = pyros_solver.solve(model = m, # doctest: +ELLIPSIS
+  ...                                  first_stage_variables = first_stage_variables,
+  ...                                  second_stage_variables = second_stage_variables,
+  ...                                  uncertain_params = uncertain_parameters,
+  ...                                  uncertainty_set = box_uncertainty_set,
+  ...                                  local_solver = local_solver,
+  ...                                  global_solver= global_solver,
+  ...                                  options = {
+  ...                                     "objective_focus": pyros.ObjectiveType.worst_case,
+  ...                                     "solve_master_globally": True
+  ...                                   })
+  ===========================================================================================
+  PyROS: Pyomo Robust Optimization Solver ...
+  ===========================================================================================
+  ...
+  INFO: Robust optimal solution identified. Exiting PyROS.
+  ...
 
   >>> # === Print results ===
   >>> single_stage_final_objective = pyo.value(results_1.final_objective_value)
-  >>> print("\n Final objective value: %s" % single_stage_final_objective)
-  >>> print("PyROS termination condition: %s" % results_1.grcs_termination_condition)
+  >>> print("Final objective value: %s" % single_stage_final_objective) # doctest: +ELLIPSIS
+  Final objective value: ...
+  >>> print("PyROS termination condition: %s" % results_1.grcs_termination_condition) # doctest: +ELLIPSIS
+  PyROS termination condition: ...
```

Comment:
You don't need to specify the `+ELLIPSIS` flag: that is enabled globally.  Also, can you include at lease the first several significant figures of the solution?  Is there a reason that the termination condition is not deterministic?

Reply:
Yes I can do that, the termination condition (assuming the same solvers) is deterministic.